src/components/DiaryCard.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {Card, CardContent , Typography , CardActions , Button } from '@material-ui/core'
+
+const useStyles = makeStyles({
+    root: {
+        width: 275,
+      },
+      title: {
+        fontSize: 24,
+      },
+      pos: {
+        marginBottom: 12,
+      },
+})
+
+
+const DiaryCard = ({tittle, subtitle, description , color}) => {
+    const classes = useStyles();
+    const[showMore , setShowMore] = useState(true);
+
+    const showDescription = showMore ? description.slice(0,100)+'...' : description;
+
+    function isShowMore() {
```

Comment:
Naming should be more suitable. Since this is a function it should be like `toggleShowMore` or any other suitable name. In here `isShowMore` is suitable for a boolean variable.